components/viewProduct/FilterDropdown.tsx

```diff
@@ -1,15 +1,16 @@
 import styled from '@emotion/styled';
-import { CategoryProps } from './ProductFilter';
 import { IcCheckbox } from '../../public/assets/icons';
 import { useRef } from 'react';
-
-export default function FilterDropdown({
-  categoryInfo,
-  isDrop,
-}: CategoryProps) {
+export interface FilterDropdownProps {
+  categoryInfo: string[];
+  isDrop: boolean;
+  isExcept: boolean;
+}
```

Comment:
이 칭구는 그러면 props에서만 쓰는게 아니니까 여기에 있는게 아니라!

`types/toy.ts`로 빼든가 아님 `type/viewProduct.ts`로 빼서 

```suggestion
export interface FilterDropdown {
  categoryInfo: string[];
  isDrop: boolean;
  isExcept: boolean;
}
```
이런식으로 쓰고 types에서 가져오는게 맞을것같아! 

viewProduct/index.ts 에는 **components**들만 있는게 맞아서!!

Reply:
이게 너무 고민이었는데,,! 감사합니당